autogpt_platform/market/market/app.py

```diff
@@ -65,7 +65,6 @@ async def lifespan(app: fastapi.FastAPI):
     allow_origins=[
         "http://localhost:3000",
         "http://127.0.0.1:3000",
-        "http://127.0.0.1:3000",
```

Comment:
Can we merge this list into one with rest_api ?
We can definitely use FRONTEND_BASE_URL .env and use that instead as @Pwuts described.

Reply:
I followed @Pwuts suggestion of using `BACKEND_CORS_ALLOW_ORIGINS` as the env var. I added a basic version to marketplace app.